python/ray/experimental/state/api.py

```diff
@@ -872,6 +1239,24 @@ def summarize_actors(
     raise_on_missing_output: bool = True,
     _explain: bool = False,
 ):
+    """Summarize the actors in cluster.
+
+    Args:
+        address: URL of the API server (e.g. http://127.0.0.1:8265).
+            If not specified, it will be retrieved from the initialized ray cluster.
+        timeout: Max timeout for requests made when getting the states.
+        raise_on_missing_output: When True, exceptions will be raised if
+            there is missing data due to truncation/data source unavailable.
+        _explain: Print the API information such as API latency or
+            failed query information.
+
+    Return:
+        :ref:`ActorSummaries <state-api-schema-actor-summaries>`
```

Comment:
This is wrong? (We currently return the dictionary that matches `ActorSummaries `)

Reply:
same for all other APIs